docs/getting_started/licensing.md

```diff
@@ -1,9 +1,70 @@
+- A product key consists of **28** alphanumeric characters separated by "-" which is provided to the user when purchasing Shift through the website: `XXXX-XXXX-XXXX-XXXX-XXXX-XXXX-XXXX`.
+
 ## Node-Locked Licensing
 
-To Be Defined
+Node-Locked Licensing in Shift requires the activation of a `PRODUCT KEY` on one single machine. As commented before, this activation process requires the user to activate the product for batch and interactive modes separately.
+
+### Interactive
+
+Whenever activating Shift for the first time, a dialog will be prompted requesting to enter a valid `PRODUCT KEY` or to launch Shift in trial mode for non-commercial purposes.
+
+To activate Shift in Node-Locked Interactive mode:
+
+  1. Launch Shift by clicking on the shift.bat file in Shift's installation folder.
+  2. A dialog will show up where to enter a valid `PRODUCT KEY` in order to enable the full license; or to continue with the 30 day trial period.
+
+<figure>
+      <img src="images/license_activation_dialog.png" alt="Shift License Activation Dialog">
+      <figcaption><b>Figure 1</b>: Activation Dialog.</figcaption>
+</figure>
+
+  3. On the text box, enter the `PRODUCT KEY` associated to your Interactive Node-Locked License.
+  4. Click on *Activate* on the bottom left corner.
+
+If the product key is invalid or connecting with the licensing servers failed, an error will be raised indicating that Shift couldn't be activated. Otherwise, Shift will be open, meaning that the license was activated successfully.
+
+> [!NOTE]
+> - This activation mode requires access to the internet for activating licenses.
+> - For deactivating licenses to switch to a different machine, please contact support.
+
+### Batch
+
+Whenever activating Shift for the first time in batch mode, a product key has to be registered previously. Activating batch mode requires the use of an executable that eases the activation process of the product.
+
+To activate Shift in Node-Locked Batch mode: 
+
+  1. Open a new terminal.
+  2. Go to `shift\tools` in the Shift installation folder.
+  2. Run the python script `batchActivate.py`.
+  3. Enter the `PRODUCT KEY`.
+  4. Shift is activated and ready to be used.
+
+### Trial
+
+Shift allows the user to use the product for **30 days** in Node-Locked Interactive mode. This means that the trial can be used using the graphical interface for one single machine at a time. 
+
+The trial period does not require any input from the user. Simply click *Continue* in the Shift activation dialog to start using the application. The trial period is registered automatically the first time that Shift is launched with internet access.
+
+Trial licenses are intended for testing and non-commercial purposes. To use Shift for commercial purposes a `PRODUCT KEY` must be purchased through Inbibo's official [website](https://inbibo.co.uk/shift#prices) and activated. See the Shift [End User License Agreement](https://inbibo.co.uk/shift-eula) for details.
```

Comment:
Only one comment related to the links to the Inbibo's webpage, I was testing it, and is not properly opening the pages for pricing and EULA. Can anyone else test this just to confirm? @Inbibo/shift

Reply:
This is what I get from our own webpage:
- https://inbibo.co.uk/shift#prices
- https://inbibo.co.uk/shift-eula

So the links are right.
What might be failing is the action associated to clicking on the link from the documentation.

Reply:
Yes, the links are correct, it's failing because the preview tool always tries to use format links from inside the documentation, not from outside.
I tested from other pages and this behaviour is persistent along the documentation pages.

Reply:
Yes, that's one of the limitations we found out earlier last week from the preview tool.